lib/RelatedResourceProviders/TalkRelatedResourceProvider.php

```diff
@@ -183,7 +192,7 @@ private function processRoomParticipant(RelatedResource $related, TalkActor $act
 
 
 	/**
-	 * @param TalkRoom $actor
+	 * @param TalkActor $actor
```

Comment:
```suggestion
```
Also in the function declaration already so not needed

Reply:
we already had a discussion about this, we keep phpdoc